### DISCUSSION TITLE
Contribution Graph stopped

### DISCUSSION THREAD

**Discussion by papastepano** [community/community#135992] [Enterprise]
Question

Hello everyone,
I am experiencing a weird issue with my contribution graph. It suddenly stopped showing contributions since I joined an organization. I read somewhere that it may be due to the policies of this organization. However, I am no longer part of this organization, and my contributions still do not show. Can you assist me in getting it fixed?
Thank you!

**Comment by romhenri** [comment]
Tell more details.

Your contributions are private in org? what contribution? in the main branch?

**Reply by papastepano** [reply]
I had all contributions to my private repositories in the graph prior joining the organization. Since I joined the organization none of my commits was in the graph (all private repos) anymore. Now I left the organization and my commits unfortunately still do not show.

**Reply by romhenri** [reply]
Review your GitHub settings to ensure no filters are affecting visibility. Verify that commits are made with an email associated with your GitHub account. 

In the corner of the commit graph, has config.

**Reply by papastepano** [reply]
I have checked 
- the graph config
- my settings 
- commit git email

prior opening this question.

**Reply by GithmiHashara** [reply]
It seems like you've already checked your settings, the graph configuration, and your commit email. Here are a few additional things to try:

Check Private Contributions Visibility: Ensure that your private contributions are set to be visible in your profile. Go to your GitHub profile settings, under the "Profile" section, and ensure the "Include private contributions on my profile" option is enabled.

Verify Commit Association: Make sure your commits are associated with the correct email address that is linked to your GitHub account. You can verify this by running git config --global user.email and ensuring it matches your GitHub account's email.

Force Profile Refresh: Sometimes, changes take a while to reflect. You can try making a new public or private commit to force GitHub to refresh your contribution graph.

Reach Out to GitHub Support: If none of the above solutions work, there might be a deeper issue, possibly related to your previous organization's policies. Contacting GitHub support could help resolve this.

**Reply by papastepano** [reply]
I have just checked and todays contribution appears. Awesome! I only lost  those contributions, while part of the organization. I can live with that
